up42/http/session.py

```diff
@@ -21,11 +21,14 @@ def request(self, *args, **kwargs) -> requests.Response:
 
 def create(
     auth: requests.auth.AuthBase,
-    create_adapter: HttpAdapterFactory = http_adapter.create,
```

Comment:
This was required to avoid mypy warning of using the initialization arguments for `http_adapter.create`